src/Lean/Server/InfoUtils.lean

```diff
@@ -91,6 +97,7 @@ def Info.stx : Info → Syntax
   | ofCompletionInfo i     => i.stx
   | ofCustomInfo i         => i.stx
   | ofUserWidgetInfo i     => i.stx
+  | ofFVarAliasInfo _      => .missing
```

Comment:
This made me briefly question the approach, but I don't see a better solution nor any problems resulting from this in practice